test/testcommon/src/main/java/io/pravega/test/common/AssertExtensions.java

```diff
@@ -45,7 +45,11 @@
     public static <T> void assertEventuallyEquals(T expected, Callable<T> eval, long timeout) throws Exception {
         long endTime = System.currentTimeMillis() + timeout;
         while (endTime > System.currentTimeMillis()) {
-            if (expected == eval.call()) {
+            if (expected == null) {
+                if (eval.call() == null) {
+                    return;
+                }
+            } else if (expected.equals(eval.call())) {
```

Comment:
Is it ok if `eval.call()` is `null` here?

Reply:
Yes, eval.call() could return null.